extensions/skyportal/static/js/components/VegaPlotZTFAlert.jsx

```diff
@@ -6,7 +6,6 @@ import embed from 'vega-embed';
 const spec = (url, jd) => ({
   $schema: "https://vega.github.io/schema/vega-lite/v4.json",
   width: 500,
```

Comment:
The 500 width looks good on larger screens, but does make it go off the paper on mobile...

Did the `width: "container"` not work out for you? Maybe we can just add a `overflowX` to the div for the plot for mobile for a quick solution? Or go back to `width: 400`?

Reply:
Yea, I spent quite a bit of time trying to make it work - tried a whole bunch of ways, but no success so far. Will address in a future PR